src/app-layout/__tests__/utils.tsx

```diff
@@ -71,6 +55,11 @@ export function describeEachThemeAppLayout(isMobile: boolean, callback: (theme:
         (useMobile as jest.Mock).mockReset();
         (useVisualRefresh as jest.Mock).mockReset();
       });
+      test('mocks applied correctly', () => {
```

Comment:
This test ensures that whenever you use this utility it renders app layout with the correct environment